styles/Vocab/readme.md

```diff
@@ -0,0 +1,3 @@
+The Vocab directory has our custom vocabulary that Vale will accept or reject.
```

Comment:
Vale errors out on words that we use often, like `automerge`, `codeowners, `URLs` and so on. 

I use `accept.txt` and `reject.txt` files to tell Vale what words we accept and reject.